DESIGN.md

```diff
@@ -117,49 +115,88 @@ ram_reserved = (256, 512, 1024) if total memory (4-7 GB, 8-16 GB, 16 GB+)
 
 > Any Replica should/would/will receive the same settings as the Primary Master, as a Replica is required to have the same system resources as the Primary Master.
 
-#### Large Reference Architecture (Compilers)
+#### Large Reference Architecture
+Master plus compilers Install
+
+##### <a name="Large-Master">Master</a>
 
-##### PuppetDB Service (pe-puppetdb)
+Calculations for the Master in a Large Ref Arch use the same algorithm as for the [Standard Reference Architecture Master](#Standard-Master) with the
+following exceptions:
 
-If this is a Compiler with PuppetDB, the following change ...
+PuppetServer on the Master will process catalog requests only for other PE Infrastructure hosts.
+While PuppetDB on the Master will be expected to handle requests from PuppetServer services on 
```

Comment:
Minor nitpick revision: "PuppetDB on the Master will be expected to handle requests from PuppetServer services on multiple Compilers that together are servicing more agents then the Standard Ref Arch. So resources on the master are transferred from PuppetServer to PuppetDB as follows"

Reply:
pushed